language/move-binary-format/src/check_bounds.rs

```diff
@@ -68,46 +68,118 @@ impl<'a> BoundsChecker<'a> {
     }
 
     fn verify_impl(&mut self) -> PartialVMResult<()> {
+        self.check_signatures()?;
+
```

Comment:
I would suggest to remove the spaces here. Might make the function more condense.

Reply:
No problem, I will do that.